### PR TITLE
Proxy contract

### PR DESCRIPTION
**Pull request by bartekryba**
Proxy contract for account contract.

Now account contract is deployed only once, and user deploys proxy contract, that takes account contract address, eth address, and chain as arguments.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Solpatium) [inline .vscode/settings.json:1]: Did you want to commit this?

File: contracts/web3_account.cairo [hunk 2]
```diff
@@ -116,7 +141,7 @@ func constructor{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_p
 end
 
 @external
-func __execute__{
+func execute{
```
Comment (Solpatium) [inline contracts/web3_account.cairo:144]: Why did you change the name?

File: eip712-starknet-account/src/web3_account_proxy.json [hunk 3]
```diff
@@ -0,0 +1 @@
...Nm5jcT4HCZVfR9UpCLmUT7K+z7qRlnMw2CURj4sRWWXQQqTQQo6WKCzbzd1RTi76y4htDnt9WQkpguP4dieMoa6hlgaMzqFu8Z4NHrIZ/PRqMKv/fnjKl8uy9X69PerutpfbSnoID9B1cbKTI/EIefRGctjfn9IfTLVIWTrvtX7UfFB61nfJQLuKjcfGjkSzljAvbOH5X1ZwZHXKVKTDPEks4tjM9l9fZ2TovrKrOvKZTnRX5bzfeCDcNfVq3yuF8DiQ1n8drihU/CVtuHdtlN1Le269igEQ51GTtqxuhCVnW/fD6Ftt7bsyhSbtjwpx4/v983u88f7+3/+P3iN48SbJgIA", "entry_points_by_type": {"CONSTRUCTOR": [{"selector": "0x28ffe4ff0f226a9107253e17a904099aa4f63a02a5621de0576e5aa71bc5194", "offset": "0x10e"}], "EXTERNAL": [{"selector": "0x0", "offset": "0x134"}], "L1_HANDLER": []}, "abi": [{"data": [{"name": "implementation", "type": "felt"}], "keys": [], "name": "Upgraded", "type": "event"}, {"inputs": [{"name": "implementation_address", "type": "felt"}, {"name": "eth_address", "type": "felt"}, {"name": "chain", "type": "felt"}], "name": "constructor", "outputs": [], "type": "constructor"}, {"inputs": [{"name": "selector", "type": "felt"}, {"name": "calldata_size", "type": "felt"}, {"name": "calldata", "type": "felt*"}], "name": "__default__", "outputs": [{"name": "retdata_size", "type": "felt"}, {"name": "retdata", "type": "felt*"}], "type": "function"}]}, "constructor_calldata": [], "type": "DEPLOY"}
```
Comment (Solpatium) [inline eip712-starknet-account/src/web3_account_proxy.json:1]: You should delete previous file, we won't have to deploy it from FE.

File: .vscode/settings.json (hunk 1 above)
Comment (bartekryba) [reply .vscode/settings.json:1]: It fixes tsconfig problem, which emerges when project is nested in other folder, like in our case.

File: contracts/web3_account.cairo (hunk 2 above)
Comment (bartekryba) [reply contracts/web3_account.cairo]: By mistake, fixed it.

File: .vscode/settings.json (hunk 1 above)
Comment (Solpatium) [reply .vscode/settings.json:1]: Right, but you could have it locally and git ignored.
